modules/calib3d/src/ippe.cpp

```diff
@@ -0,0 +1,1122 @@
+/* Copyright 2017 Toby Collins
+Redistribution and use in source and binary forms, with or without
```

Comment:
Could you please use OpenCV short header about this? (see `core/src/softfloat.cpp`)